src/main/java/command/UserInput.java

```diff
@@ -0,0 +1,130 @@
+package command;
+import java.util.Scanner;
+
+import task.Deadline;
+import task.Event;
+import task.Task;
+import task.ToDo;
+
+import java.util.InputMismatchException;
+import java.util.List;
+
+public class UserInput {
+    public static void parseInput(String input, Scanner in, List<Task> taskList) {
+
+        int taskID;
+
+        switch (input.toLowerCase()) {
+
+        case "list":
+            printLists(taskList);
+            break;
+
+        case "bye":
+            break;
+
+        case "mark":
+            taskID = in.nextInt();
+            executeMark(taskList, taskID);
+            break;
+
+        case "unmark":
+            try {
+                taskID = in.nextInt();
+                executeUnmark(taskList, taskID);
+            } catch (InputMismatchException e) {
+                System.out.println("A number please");
+            }
+            break;
+
+        case "todo":
+            input = in.nextLine();
+            parseToDo(input, taskList);
+            break;
+
+        case "deadline":
+            input = in.nextLine();
+            parseDeadline(input, taskList);
+            break;
+
+        case "event":
+            input = in.nextLine();
+            parseEvent(input, taskList);
+            break;
+
+        default:
+            System.out.println("Please input a valid command");
+            break;
+        }
+    }
+
+    private static void printLists(List<Task> taskList) {
+
+        for (int i = 1; i <= taskList.size(); i += 1) {
+            System.out.print(i + ".");
+            System.out.println(taskList.get(i - 1));
+        }
+    }
+
+    private static void executeMark(List<Task> taskList, int taskID) {
```

Comment:
consider naming this `markAsComplete` or `markTask`, `executeMark` does not quite describe the function well. 

https://nus-cs2113-ay2324s2.github.io/website/se-book-adapted/chapters/codeQuality.html#use-name-to-explain